### PR TITLE
docs(elastic-for-graph): Add migrating from neo4j to elastic instructions

### PR DESCRIPTION
**Pull request by gabe-lyons**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docs/how/migrating-graph-service-implementation.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+```aidl
+docker volume rm datahub_neo4jdata
+```
+
+This prevents your DataHub instance from coming up in neo4j mode in the future.
+
+## Helm
+
+First, follow the [deployment helm guide](../../datahub-kubernetes/README.md#components) to set up your helm deployment
```
Comment (dexter-mh-lee) [inline docs/how/migrating-graph-service-implementation.md:40]: Can you link to the line in the quickstart-without-neo4j.values.yaml that sets graph_service to elasticsearc?

File: docs/how/migrating-graph-service-implementation.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Migrate Graph Service Implementation to Elasticsearch
+
+We currently support either Elasticsearch or Neo4j as backend implementations for the graph service. We recommend
+Elasticsearch for those looking for a lighter deployment or do not want to manage a Neo4j database.
+If you started using Neo4j as your graph service backend, here is how you can migrate to Elasticsearch.
+
+## Docker-compose
+
+If you are running your instance through docker locally, you will want to spin up your Datahub instance with
```
Comment (dexter-mh-lee) [inline docs/how/migrating-graph-service-implementation.md:9]: Can you also add the command to do this using datahub cli?

**Review by dexter-mh-lee — APPROVED** [review]
Overall LGTM! Minor comments

File: docs/how/migrating-graph-service-implementation.md (hunk 2 above)
Comment (gabe-lyons) [reply docs/how/migrating-graph-service-implementation.md:9]: updated

File: docs/how/migrating-graph-service-implementation.md (hunk 1 above)
Comment (gabe-lyons) [reply docs/how/migrating-graph-service-implementation.md]: updated

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**